starlette/responses.py

```diff
@@ -294,7 +294,7 @@ async def __call__(self, scope: Scope, receive: Receive, send: Send) -> None:
             }
         )
         if self.send_header_only:
-            await send({"type": "http.response.body", "body": b"", "more_body": False})
+            await send({"type": "http.response.body"})
```

Comment:
Actually I *do* still think it's probably worth this being `send({"type": "http.response.body", "body": b"", "more_body": False})` since that's more explicit. (Even though it's not required now.)

Reply:
ok I put it back :)